package.json

```diff
@@ -22,7 +22,7 @@
     "add-suffix": "node ./node_modules/npm-version-suffix/run-add-suffix.js",
     "remove-suffix": "node ./node_modules/npm-version-suffix/run-remove-suffix.js",
     "prepare": "yarn build && husky install",
-    "test": "jest",
+    "test": "jest --coverage",
```

Comment:
This is not needed

Reply:
ah yes! with the flag set in the jest config, this is not needed. I've removed it now.